plugins/hypervisors/vmware/pom.xml

```diff
@@ -60,14 +60,6 @@
             <artifactId>resolver</artifactId>
             <version>20050927</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.axis</groupId>
-            <artifactId>axis</artifactId>
```

Comment:
I'm not sure about this, why not upgrading to axis2 here? Worried about this: https://github.com/apache/cloudstack/blob/main/plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareContextFactory.java#L48 and https://github.com/apache/cloudstack/blob/main/plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareSecondaryStorageContextFactory.java#L35

Reply:
From https://axis.apache.org/axis2/java/core/docs/migration.html doesn't look like this would cause an issue.